code/modules/projectiles/guns/energy/sniperrifle.dm

```diff
@@ -10,7 +10,7 @@
 	projectile_type = /obj/item/projectile/beam/sniper
 	slot_flags = SLOT_BACK
 	charge_cost = 300
-	fire_delay = 35
+	fire_delay = 12
```

Comment:
I think valk should keep it's very high fire delay.

Reply:
Valk sucks as it is.  Don't see why it shouldn't be more on par with the lightfall.

Reply:
> Valk sucks as it is.  Don't see why it shouldn't be more on par with the lightfall.

Because it is a sniper weapon and has high damage

Reply:
Ah yes , the highly lethal energy rifle that has a maximum potential of 250 damage that is not enough to kill anyone should remain as it is , while an AK .30 with a lower rarity can easily kill a person  with a mag dump.

Reply:
> Ah yes , the highly lethal energy rifle that has a maximum potential of 250 damage that is not enough to kill anyone should remain as it is , while an AK .30 with a lower rarity can easily kill a person with a mag dump.

valk sucks but it doesnt need to become a machinegun

Reply:
I'll agree with TheShown and Valo, it has high damage and doesn't need fire_delay decrease

Reply:
> man paco gets a fire delay reduction and MK58 dont. sad

if this pr gets closed because SPCR is too stubborn to change, i will just re-do it but including the mk58, and any other gun you think might deserve it